hedyorg/hedy
# [REFACTORING] Changes Parsons YAML structure to new format

**Description**
This PR built upon #3697 to improve the Parsons YAML structure for all files.

**Fixes**
This PR has no explicit related issue, but is further work for https://github.com/hedyorg/hedy/issues/3677.


**How to test**
As we validate most of the code by caching this PR should be fine once the tests pass. To make sure, try the Parsons in several languages and verify that it still works as expected.
